src/components/AddressSearch/index.tsx

```diff
@@ -264,9 +157,9 @@ function AddressSearch({
             city: locality || postalTown || sublocality || cityAutocompleteFallback,
             zipCode,
 
-            lat: lodashGet(details, 'geometry.location.lat', 0),
-            lng: lodashGet(details, 'geometry.location.lng', 0),
-            address: autocompleteData.description || lodashGet(details, 'formatted_address', ''),
+            lat: details.geometry.location.lat ?? 0,
+            lng: details.geometry.location.lng ?? 0,
+            address: autocompleteData.description ?? details.formatted_address ?? '',
```

Comment:
Is it possible that `autocompleteData.description` will be `''`? If so it will be better to use ` || ` as it initially was.

Reply:
It is, done